client/mysqlslap.c

```diff
@@ -282,6 +284,27 @@ static long int timedif(struct timeval a, struct timeval b)
     return s + us;
 }
 
+void sleep_us(ulong tm)
+{
+#ifdef _WIN32
+    Sleep((DWORD) tm / 1000);
+#elif defined(HAVE_NANOSLEEP)
```

Comment:
We are missing this check in CMakeFiles. This doesn't ever get defined in client directory.